src/main/scala/service/UserRepo.scala

```diff
@@ -2,30 +2,42 @@ package service
 
 import Dao.DAO
 import models.Users
-import java.util.UUID
+
+import java.sql.{ResultSet, SQLException}
+import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
-import scala.collection.mutable.ListBuffer
+import scala.util.{Failure, Success, Try}
 
 class UserRepo(userDB: DAO) {
 
-  def addUser(user: Users): Future[String] = {
+  def addUser(user: Users): Future[String] = Future {
     userDB.addUser(user)
+
+    "Data inserted successfully!"
   }
 
-  def getById(userId: UUID): Future[Option[Users]] = {
-    userDB.getById(userId)
+  def getById(userId: Int): Future[ResultSet] = {
+    Try(userDB.getById(userId)) match {
+      case Success(resultSet) => resultSet
+      case Failure(_) => throw new SQLException
+    }
   }
 
-  def getAll: Future[ListBuffer[Users]] = {
-    userDB.getAll
+  def getAll: Future[ResultSet] = {
+    Try(userDB.getAll) match {
```

Comment:
Code is duplicated here also. So move this also to the method and call that method here and on line 20.

Reply:
It is not in my code(on my local), I have changed it to Future[List[Users]].

Reply:
Oh, okay sorry you mean the duplicacy, I will resolve it.